src/main/java/io/github/jbellis/brokk/gui/GitPullRequestsTab.java

```diff
@@ -1104,6 +1115,62 @@ private void disablePrButtons() {
         diffPrButton.setEnabled(false);
         viewPrDiffButton.setEnabled(false);
         openInBrowserButton.setEnabled(false);
+        capturePrDiffButton.setEnabled(false);
+    }
+
+    private void captureSelectedPrDiff() {
+        int selectedRow = prTable.getSelectedRow();
+        if (selectedRow == -1 || selectedRow >= displayedPrs.size()) {
+            return;
+        }
+        GHPullRequest pr = displayedPrs.get(selectedRow);
+        logger.info("Capturing diff for PR #{}", pr.getNumber());
+
+        // Check if instructions are empty and populate with PR title + review prompt if needed
+        SwingUtilities.invokeLater(() -> {
+            String currentInstructions = chrome.getInstructionsPanel().getInstructions();
+            if (currentInstructions == null || currentInstructions.trim().isEmpty()) {
```

Comment:
Not auto-populating the instructions if user already entered some text, I hope that makes sense (i.e. user maybe wants the PR diff for something else other than review)?